var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -397,4 +398,8 @@ def nvariant_bool(feature):
                 )
             )
 
+        if '+debug' in spec:
+            cmake_args.append('-DCMAKE_BUILD_TYPE=Debug')
```

Comment:
You should not set CMAKE_BUILD_TYPE as that is already set with the build_type variant. So here just set any other debug flags such as VTK_DEBUG_LEAKS